README.md

```diff
@@ -344,6 +344,19 @@ const { data: app } = await requestWithAuth(
       Used for internal logging. Defaults to <a href="https://developer.mozilla.org/en-US/docs/Web/API/console"><code>console</code></a>.
     </td>
   </tr>
+  </tr>
+    <th align=left>
+      <code>options.request.parseSuccessResponseBody</code>
+    </th>
+    <td>
+      <code>boolean</code>
+    </td>
+    <td>
+      If set to <code>false</code> the request will return a Stream as response.data for successful requests (e.g. <code>200 OK</code> status code).
+      This option may be useful when downloading files from the GitHub API.
+      Defaults to <code>true</code>.
```

Comment:
I think it's `response.body`? 

Maybe phrase it like this

```suggestion
      If set to <code>false</code> the returned `response` will be passed through from `fetch`. This is useful to stream response.body when downloading files from the GitHub API.
```

I would also leave out the`Defaults to <code>true</code>.` bit because I'd say by default we don't set it to anything. In future we might as well support setting it to a function.

Reply:
the thing is that request.js is changing body to data. Maybe I should change it to parseSuccessResponseData?

Reply:
Where did you see that we were changing `body` to `data`? The `data` parameter was removed, https://github.com/octokit/request.js/pull/604